capstone/src/main/java/com/step902020/capstone/Event.java

```diff
@@ -30,17 +30,27 @@
 
   @Field(name="eventLongitude")
   double eventLongitude;
+
+  @Field(name="foodAvaliable")
+  boolean foodAvaliable;
+
+  @Field(name="requiredFee")
+  boolean requiredFee;
   
   @Reference
   List<Review> reviews;
+ 
 
-  public Event(Organization organization, String eventTitle,String eventDateTime, String eventDescription, double eventLatitude, double eventLongitude) {
+  public Event(Organization organization, String eventTitle,String eventDateTime, String eventDescription, double eventLatitude, double eventLongitude, boolean foodAvaliable, boolean requiredFee) {
```

Comment:
Non-actionable musings follow.

At some point, might be that a no-argument constructor with setters for each field will be easier to manage than a constructor with lots of fields. Or... you could pass filters in a key-value map, and then set the individual fields in the constructor body.

But I am not sure how this will mesh with this being a data entity; needs to be tested and/or researched.

Reply:
I was actually beginning to look into this. I thought using setters would be better long term, but I'll dig a little deeper and see what I can find